locales/base.json

```diff
@@ -38,6 +38,7 @@
     "immortalx": "Trade crypto with low fees directly from your wallet, with up to 50x leverage",
     "sarafu": "Create, use and explore Community Asset Vouchers with Grassroots Economics",
     "hedgey": "Empower your team and investors with on-chain token vesting",
-    "talentprotocol": "Where builders commit to goals, share their progress and find the support they need to succeed"
+    "talentprotocol": "Where builders commit to goals, share their progress and find the support they need to succeed",
+    "onrampmoney": "The financial platform for buying/selling crypto and accepting crypto payments.Convert fiat to web3 money"
```

Comment:
The description should be 50 characters or less and adhere to the other standards listed - [description requirements](https://docs.valora.xyz/listing/adding-new-dapps#required-changes).